python/az/aro/azext_aro/custom.py

```diff
@@ -153,23 +163,40 @@ def aro_delete(cmd, client, resource_group_name, resource_name, no_wait=False):
             type='virtualNetworks',
             name=master_parts['name'],
         )
+        
+        route_tables = get_route_tables_from_subnets(cmd.cli_ctx, worker_subnets | {master_subnet})
```

Comment:
nit: as non python dev when syntax like this is used it makes my brain :exploding_head: